prov/util/src/util_attr.c

```diff
@@ -434,7 +434,7 @@ int ofi_check_domain_attr(const struct fi_provider *prov, uint32_t api_version,
 	    strcasecmp(user_attr->name, prov_attr->name)) {
 		FI_INFO(prov, FI_LOG_CORE, "Unknown domain name\n");
 		FI_INFO_CHECK(prov, prov_attr, user_attr, name,
-			      FI_TYPE_DOMAIN_ATTR);
+			      FI_TYPE_NAME);
```

Comment:
I think we can just drop the FI_INFO_CHECK macro and just merge the names with the previous FI_INFO call